profiler/log.go

```diff
@@ -0,0 +1,42 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2022 Datadog, Inc.
+
+package profiler
+
+import (
+	"fmt"
+	"log"
+
+	"gopkg.in/DataDog/dd-trace-go.v1/ddtrace"
+	"gopkg.in/DataDog/dd-trace-go.v1/internal/version"
+)
+
+var prefixMsg = fmt.Sprintf("Datadog Tracer %s", version.Tag)
+
+type logger struct {
+	l ddtrace.Logger
+}
+
+func newLogger(l ddtrace.Logger) *logger {
+	return &logger{l: l}
+}
+
+func (l *logger) Info(format string, a ...interface{})  { l.msg("", format, a...) }
+func (l *logger) Error(format string, a ...interface{}) { l.msg("", format, a...) }
+func (l *logger) Warn(format string, a ...interface{})  { l.msg("", format, a...) }
```

Comment:
Why is the `lvl = ""` for all of the `msg()` calls above?

Also: Wouldn't it be nicer to put this `logger` implementation into internal/log and reuse it here? Or is there a specific reason for the code duplication? (It's not a lot of duplication, so I'm not too worried, just curious).